frontend/public/components/mounted-vol.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import * as _ from 'lodash-es';
+
+import {
+  getVolumeType,
+  getVolumeLocation,
+  getVolumeMountPermissions,
+  getVolumeMountsByPermissions,
+} from '../module/k8s/pods';
+import {
+  VolumeIcon,
+  ResourceIcon,
+  EmptyBox,
+  SectionHeading,
+} from './utils';
+
+const ContainerLink = ({name, pod}) => <span className="co-resource-link co-resource-link--inline">
+  <ResourceIcon kind="Container" />
+  <Link to={`/k8s/ns/${pod.metadata.namespace}/pods/${pod.metadata.name}/containers/${name}`}>{name}</Link>
+</span>;
+
+const Volume = ({volumeMounts, pod}) => {
+  const kind = _.get(getVolumeType(volumeMounts.volume), 'id', '');
+  const loc = getVolumeLocation(volumeMounts.volume);
+  const name = volumeMounts.name;
+  const mountPermissions = getVolumeMountPermissions(volumeMounts);
+
+  return <div>
+    {volumeMounts.mounts.map((m, i) => <React.Fragment key={i}>
+      <div className="row">
+        <div className="col-lg-2 col-md-3 col-sm-4 col-xs-5">{name}</div>
+        <div className="col-lg-2 col-md-3 col-sm-5 col-xs-7 co-break-word">{_.get(m, 'mountPath', '-')} </div>
+        <div className="col-lg-2 col-md-2 col-sm-3 hidden-xs co-break-word">{_.get(m, 'subPath', '-')} </div>
+        <div className="col-lg-2 col-md-2 hidden-sm hidden-xs">
```

Comment:
We might revisit the order we drop the columns. I feel like the kind of mount is more important than some of the other values we show at `sm` screen widths.